include/revng/Storage/DummyStorageClient.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "llvm/ADT/StringSet.h"
+
+#include "revng/Storage/StorageClient.h"
+#include "revng/Support/Debug.h"
+
+namespace revng::detail {
+
+class DummyStorageClient : public StorageClient {
```

Comment:
Docs, I expect it will not do much, but not much *how*?

Reply:
Added docs